utils/utils.jsx

```diff
@@ -2155,3 +2157,9 @@ export function stringToNumber(s) {
 
     return parseInt(s, 10);
 }
+
+export function shouldProcessApps(state) {// eslint-disable-line no-unused-vars
+    // TODO uncomment when featur flag is in place
+    //return getConfig(state)?.['FeatureFlagApps' as keyof Partial<ClientConfig>];
+    return true;
+}
```

Comment:
Heads up that I used that stub function in https://github.com/mattermost/mattermost-webapp/pull/7474. If you change the signature, you will also have to change it in `actions/marketplace.ts`.